core/util/google.py

```diff
@@ -30,7 +30,7 @@ def __init__(self, q, limit=1, count=10, google_api=None, google_cx=None):
 		"""
 		self.framework = main.framework
 		self.q = q
-		self.agent = 'Mozilla/5.0 (X11; Linux x86_64; rv:68.0) Gecko/20100101 Firefox/68.0'
+		self.agent = 'Mozilla/5.0 (X11; Linux x86_64; rv:86.0) Gecko/20100101 Firefox/86.0'
```

Comment:
also https://www.keycdn.com/blog/web-crawlers

Reply:
![image](https://user-images.githubusercontent.com/42383989/113205744-3f4cb300-928c-11eb-8c14-3f93d712838f.png)
Nowhere in Maryam do we use this. do you think giving a User-agent corresponding to this will be helpful?

Reply:
yes, we can use some randomness here by creating try and exception block in requests (which makes our tool better 😎)
advantage:
  - get the restricted content
  - get randomness in request
 
dis-advantage:
  - only one request slow result which is almost negligible in good connection
👍